docs/en/ingest-management/release-notes/release-notes-8.18.asciidoc

```diff
@@ -33,6 +33,45 @@ Also see:
 [[release-notes-8.18.7]]
 == {fleet} and {agent} 8.18.7
 
+[discrete]
+[[known-issues-8.18.7]]
+=== Known issues
+
+[[known-issue-2978-8-18-7]]
+.Failed upgrades leave {agent} stuck until restart
+[%collapsible]
+====
+
+This known issue applies to {agent} 8.18.7 and 9.0.7. {agent} versions 8.19.x and 9.1.x are not affected.
+
+On September 17, 2025, a known issue was discovered that can cause {agent} upgrades to get stuck if an upgrade attempt fails early. This happens because the coordinator's overrideState remains set, leaving the agent in a state that appears to be upgrading.
```

Comment:
```suggestion
On September 17, 2025, a known issue was discovered that can cause {agent} upgrades to get stuck if an upgrade attempt fails under specific conditions. This happens because the coordinator's overrideState remains set, leaving the agent in a state that appears to be upgrading.
```

I think it's important to emphasize this problem does not happen for any upgrade failed, like the most common cases where the download fails.